packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -143,9 +146,17 @@ function CustomAddressBanner({
 
   return (
     <div
+      style={{
+        backgroundColor: isOrbitChain
+          ? // create a lighter color from defined config colors for Orbit chains
+            lightenColor(primaryColor, 60)
```

Comment:
We defined some of the lighter backgrounds for the L1 and L2 networks, but if we want to be scalable with new Orbit chains we cannot just hardcode more colors here.

<img width="580" alt="Screenshot 2024-01-12 at 16 26 45" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/47932951/bba07436-4a0b-402e-9aca-9b33ad1aa6fd">

Adding a tertiary might make sense but its only used in this instance so it might be slightly more annoying to add configs for new Orbit chains. With this util we have an automated fix that gives us a lighter shade

Reply:
much nicer, updated